src/netspresso_trainer/cfg/training.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+
+from omegaconf import MISSING, MissingMandatoryValue
+
+
+@dataclass
+class ScheduleConfig:
+    seed: int = 1
+    opt: str = "adamw"
```

Comment:
P4) I think it would be good to handle optimizer & scheduler as Enum, what do you think? ([Optimizer support list in np-trainer docs](https://nota-netspresso.github.io/netspresso-trainer/components/training/#supporting-optimizers)) 

```
ex)
Optimizer.Adamw
Optimizer.Adam
...

Scheduler.Cosine
...
```

Reply:
@Only-bottle 

We may internally construct `Enum`, but I personally believe that it doesn't need to be exposed to the user.
Most training repositories just keep it as string nickname in their training script.
It might also be the problem if the user uses trainer with the yaml configuration, which does not support pythonic `Enum` itself.

Reply:
@illian01 How do you think?

Reply:
Insider discussion: no change, keep it as is.